GOVERNANCE.md

```diff
@@ -0,0 +1,205 @@
+# Eiffel Community Governance
+
+Eiffel is a protocol for technology agnostic machine-to-machine communication in continuous integration and delivery pipelines, aimed at securing scalability, flexibility and traceability.
+Eiffel is based on the concept of decentralized real time messaging, both to drive the continuous integration and delivery system and to document it.
+
+Governance of the Eiffel Community is structured according to this document.
+This structure in intended to provide a healthy, sustainable and attractive community for Eiffel development and collaboration.
```

Comment:
typo: is

Reply:
Can you clarify a bit? I don't see the typo.